### DISCUSSION TITLE
Question about opensource project license

### DISCUSSION THREAD

**Discussion by rabbitism** [PrismLibrary/Prism#3264] [General] [answered]
Hello Prism team,

  I am a developer for open source project Ursa, which is an open source Avalonia control library. We have an extension pack for integrating our overlay popup mechanism with Prism. From license perspective, are we allowed to provide Prism 9 support? Will upgrating to Prism 9 impact our MIT license? 

  Best Regards.

Detail of the extension: https://github.com/irihitech/Ursa.Avalonia/tree/main/src/Ursa.PrismExtension

**Comment by brianlagunas** [comment, marked as answer]
If you include Prism 9 as a dependency, you will be exposing your users to the Prism licensing requirements. It will be important that you add Prism's license to your extension that uses Prism so that your users are informed that they need to consider Prism's licensing when using that specific extension.
